### ISSUE TITLE
Error creating SSL Content () (99)

### ISSUE THREAD

**Issue by EdwardD20** [QuiteRSS/quiterss#1582]
I am unable to add certain sites, always with the same error. MacOS 13.4 if it matters.
<img width="399" alt="Screenshot 2023-06-11 at 6 19 38 PM" src="https://github.com/QuiteRSS/quiterss/assets/5359857/df368ad3-e7af-4491-b082-2d4cbb3bf179">

**Comment by hamzamalikpurelogics** [comment]
Facing same Issue

**Comment by hamzamalikpurelogics** [comment]
@EdwardD20 Did you get any solution for this problem yet?

**Comment by EdwardD20** [comment]
Nope

**Comment by EdwardD20** [comment]
I should also add this has been going on for several months. I don’t expect a quick solution, but if I didn’t report it, they wouldn’t know about it.

Ed

**Comment by hamzamalikpurelogics** [comment]
That's correct. I have tried every possible solution too but all in vain. Let's hope they can look into this matter ASAP

**Comment by EdwardD20** [comment]
I've decided it's all HTTPS feeds that fail.

**Comment by EdwardD20** [comment]
I've realized that QuiteRSS isn't supported anymore.

I'm trying https://netnewswire.com (Mac version)

**Comment by vitusson** [comment]
_I've decided it's all HTTPS feeds that fail._

I  don't see that.
I have >100 feeds with https and everyone  works well so far.

This https://www.newsnationnow.com/feed doesn't work with QRss though. But it doesn't work in newsboat or Firefox's Feedbro either, so I guess at least for this URL the problem is not on the clients side.
